src/helpers.ts

```diff
@@ -29,7 +29,8 @@ export function makeAddJob(
           job_key => $6::text,
           priority => $7::int,
           flags => $8::text[],
-          job_key_mode => $9::text
+          job_key_mode => $9::text${!now ? "" : `,
+          now => $10::timestamptz`}
```

Comment:
Let's make this less fluid:

```suggestion
          job_key_mode => $9,
          now => coalesce($10::timestamptz, now())
```